core-plugin/src/com/google/gct/idea/debugger/ui/CloudDebugHistoricalSnapshots.java

```diff
@@ -405,6 +408,7 @@ public void run() {
           break;
         }
         if (rowForPopup != -1) {
+          UsageTrackerService.getInstance().trackEvent(GctTracking.CATEGORY, GctTracking.CLOUD_DEBUGGER, "snapshot.received", null);
```

Comment:
From what I could tell, a new snapshot received happens in here.  Other changes are still processed by this method.